static/scss/geosuggest.scss

```diff
@@ -0,0 +1,69 @@
+// https://github.com/ubilabs/react-geosuggest/blob/master/src/geosuggest.css
```

Comment:
is this file copied from a library? if so, you should be able to just `require` it, in the way we do most of the rest of the library CSS (see `static/js/entry/style.js` for an example).